bin/gdf_distance

```diff
@@ -279,9 +279,12 @@ if __name__ == "__main__":
         norms = piernik_gdf_compare(sys.argv[1], sys.argv[2])
         tot_norm = 0.
         failed = False
+        longestkey = 0
+        for i in norms.keys():
+            longestkey = max(longestkey, len(i))
```

Comment:
How about more pythonic: `longestkey = len(max(norms, key=len))` instead of this for loop?

Reply:
Feel free to convert such fortranisms into pythonic code :-)